kandinsky/fonts/rasterizer.c

```diff
@@ -123,12 +123,15 @@ int main(int argc, char * argv[]) {
   }
 
   int glyph_width = maxWidth-1;
+  if (glyph_width == 9) { glyph_width =  10;}
+  if (glyph_width == 8) { glyph_width =  7;}
```

Comment:
This is... strange. Can you at least explain the problem in a comment ?

Reply:
yes the problem was with the fonts dimensions, for example the small font is 7 by 13 but for some reason when I compiled it returned this: 

Error: Error: the glyph size (7x13@4bpp) cannot fit in an integral number of bytes
make: *** [kandinsky/Makefile:94: output/release/device/n0110/kandinsky/fonts/SmallFontExtended.cpp] Error 255

I had to artificially change it to 7 by 14.

Reply:
Did you try with other values ?

Reply:
yes, the problem is the combination of the two numbers not themselves.